src/patternfly/components/Title/docs/code.md

```diff
@@ -0,0 +1,16 @@
+## Overview
+When you don't want to use the Content component and just want to use single title styles.
+
+## Usage
+
+| Class | Applied | Outcomete |
```

Comment:
Is "Outcomete" a typo? I'm assuming it's meant to be "Outcome"

Reply:
yup :) I just noticed in my list code doc too. @andresgalante it's coming from your https://gist.githubusercontent.com/andresgalante/cad24b6053ac7d0f8f1c6de89d5ace67/raw/2c28355502a407186aa9a744bfdb490fc331f4fe/test.md